Makefile

```diff
@@ -52,9 +52,22 @@ help: ## Display this help.
 
 ##@ Development
 
+GINKGO_VERSION ?= $(shell go list -m -f '{{.Version}}' github.com/onsi/ginkgo/v2)
+INTEGRATION_TARGET ?= ./test/integration/...
+
+GINKGO = $(shell pwd)/bin/ginkgo
+.PHONY: ginkgo
+ginkgo: ## Download ginkgo locally if necessary.
+	test -s $(LOCALBIN)/ginkgo || \
+	GOBIN=$(LOCALBIN) go install github.com/onsi/ginkgo/v2/ginkgo@$(GINKGO_VERSION)
+
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=controller-manager-role crd:maxDescLen=0,generateEmbeddedObjectMeta=true webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) \
+	rbac:roleName=controller-manager-role output:rbac:artifacts:config=config/rbac/controller-manager \
```

Comment:
In the past, we maintain this manually. some internal users want to use the component separately. so we change the folder structure to component based. If we use `config/rbac/controller-manager`. other components like orchestration will land into same subfolder?

Reply:
Revert since kubebuilder can't separate the roles right now, and maybe unable in a long time. I added the roles manually as well.